.github/workflows/ci.yml

```diff
@@ -12,22 +12,18 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        jdk: [graalvm@19.2.1, adopt@1.11]
+        jdk: [graalvm@21.1.0, adopt@1.11]
     name: Test ${{ matrix.os }} -- ${{ matrix.jdk }}
     steps:
       - uses: actions/checkout@v1
         with:
           submodules: true
-      - uses: olafurpg/setup-scala@v7
+      - uses: olafurpg/setup-scala@v12
         with:
           java-version: ${{ matrix.jdk }}
-      - uses: actions/setup-node@v1
```

Comment:
If you want this back in, let me know, but I couldn't see a clear reason why it was included. So I just removed it.